.circleci/config.yml

```diff
@@ -51,7 +51,10 @@ jobs:
 workflows:
   main:
     jobs:
-      - validate
+      - validate:
+          filters:
+            tags:
+              only: "/^v.*/"
       - publish_dev:
           requires: ["validate"]
```

Comment:
```suggestion
      - publish_dev:
          requires: ["validate"]
          filters:
            tags:
              only: "/.*/"
```

Reply:
I think only `publish_stable` is intended to push tagged versions and `publish_dev` is intended to push development branches (looking at the scripts in `utils` it looks like it at least).